src/style.css

```diff
@@ -0,0 +1,108 @@
+/* #region html */
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-display: swap;
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-display: swap;
+  font-style: normal;
+  font-weight: 700;
+}
+
+* {
+  box-sizing: border-box;
+  padding: initial;
+
+  font-family: "Avenir", sans-serif;
```

Comment:
move padding to elements where you use it, do not set for all (`*`) elements

move font-family to `html` selector

Reply:
ok! well received